Libraries/LibMarkdown/Text.h

```diff
@@ -53,7 +54,7 @@ class Text final {
     const Vector<Span>& spans() const { return m_spans; }
 
     String render_to_html() const;
-    String render_for_terminal() const;
+    String render_for_terminal(size_t view_width = 0) const;
```

Comment:
Does text need the width too?

(Note that Text is not a Block subclass)

Reply:
I suppose not, but same-looking function with different arguments is kinda weird too?

Reply:
Yes; but at the same time it's unclear how text would use this info. Text, in CSS terms, is `inline`, unlike blocks which are `block`. It's clear what a viewport width means for blocks — it's the width they should take up (and adjust their height based on that). It's not clear what it means for text, because it can wrap between lines and whatnot.

I'm not saying you should remove it, but perhaps you should define a clearer semantics of what this argument means here.

Reply:
well it really means nothing for Text, I guess its existence is more confusing than not.
removing it probably makes more sense.